controllers/ansibletest_controller.go

```diff
@@ -110,7 +110,7 @@ func (r *AnsibleTestReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	runningJobName := r.GetJobName(instance, externalWorkflowCounter-1)
 	err = r.Client.Get(ctx, client.ObjectKey{Namespace: instance.GetNamespace(), Name: runningJobName}, runningAnsibleJob)
```

Comment:
is it ok to proceed if `err !=nil` ? should there be a check for if err != nil and maybe special handling for IsNotFound if it is ok to proceed?

Reply:
Yes, better processing of the err would be better here. Thank you! I've added an if-statement that deals with the error. `IsNotFound` error is acceptable in this stage of the control loop as the job gets created in a later stage of the control loop.